README.rst

```diff
@@ -151,6 +151,12 @@ a light weight dependency.
 ``dask.bag`` depends on ``toolz`` and ``dill``.
 
 
+Notebooks
+---------
+
+Check example notebooks using dask at https://notebooks.anaconda.org/dask/.
```

Comment:
I don't know.  I would leave them in the top level until need forces us to
do otherwise.

On Fri, Aug 14, 2015 at 8:14 AM, Christine Doig <notifications@github.com>
wrote:

> In README.rst
> <https://github.com/ContinuumIO/dask/pull/562#discussion_r37086136>:
>
> > @@ -151,6 +151,12 @@ a light weight dependency.
> >  ``dask.bag`` depends on ``toolz`` and ``dill``.
> >
> >
> > +Notebooks
> > +---------
> > +
> > +Check example notebooks using dask at https://notebooks.anaconda.org/dask/.
>
> Sounds good. Done.
>
> Do you think the repo should have a folder with the notebooks or just
> leave them in the top repo directory for now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/dask/pull/562/files#r37086136>.
>